modules/imaging_uploader/jsx/log_panel.js

```diff
@@ -0,0 +1,175 @@
+/* global UploadProgress */
+
+class LogPanel extends React.Component {
```

Comment:
@driusan Should we adapt this approach for all of our React components? Or you just want me to rename it to `logPanel.js`